### ISSUE TITLE
Cant backup DB table name longer than 64 chars

### ISSUE THREAD

**Issue by danyj** [ThemeFuse/Unyson#3530]
This is not Unyson issue but issue nonthe less, 

DB table chars max is 64 

this is   65 + 
![dwqdwq](https://user-images.githubusercontent.com/2340172/43159737-78bad43c-8f83-11e8-85fd-7de2e7a76881.jpg)

DB installs stop in because backup cant proceed

**Comment by andreiglingeanu** [comment]
Hm, that is super strange. Do you think this is something Unyson specific?

I think that's a limit imposed by [mysql](https://stackoverflow.com/a/23774578/3220977), I don't know if there's a way to overcome it.

**Comment by danyj** [comment]
Yes it is mysql limit thus I said not Unyson issue. I advised customer to try to prefix tables with less chars but I think is one of those auto installs where they use domain name as prefix. There are numerous server issues also so is 100% hosting issue. But we should be aware of it. 

Unyson adds 6 chars as prefixes _fwbkp_ than domain 12 and + 48 db table name so u have 65 -66 chars

**Comment by andreiglingeanu** [comment]
Oh, the trouble is that you're dealing with WOO tables, and they definitely cannot change easily. 

Maybe `backup` extension could shorten the `_fw_bk` prefix, just to be fully compatible with WOO?

cc. @ViorelEremia

**Comment by danyj** [comment]
> could shorten the _fw_bk

not sure if it would help , if the customer has domain name that 14 chars long it would be same thing again, 

wp_mydomain)name_   was used as prefix and that took away most chars 

wp_dm_  would have be sufficient
